### PR TITLE
add mjs binaries and approvers to list

### PR DESCRIPTION
**Pull request by erwinmombay**
now that we report mjs files we also need to add it to the approvers list

### CODE REVIEW

**Comment by erwinmombay** [comment]
@danielrozenberg this will work out of the box correct?

File: build-system/tasks/bundle-size/APPROVERS.json [hunk 1]
```diff
@@ -3,20 +3,40 @@
     "approvers": ["ampproject/wg-performance"],
     "threshold": 0.1
   },
+  "dist/v0.mjs": {
```
Comment (rcebulko) [inline build-system/tasks/bundle-size/APPROVERS.json:6]: @danielrozenberg How hard would it be to allow using globs/regex for these? It seems like there will rarely be a case where the approvers should be any different between the two

Comment (danielrozenberg) [reply build-system/tasks/bundle-size/APPROVERS.json:6]: Probably easy, dunno if it's worth bothering though given that nobody edits this file 🙃

**Review by danielrozenberg — APPROVED** [review]

**Review by rcebulko — APPROVED** [review]

**Comment by erwinmombay** [comment]
@kevinkimball do you mind reviewing this for performance approval

**Review by kristoferbaxter — APPROVED** [review]

**Pull request merged**